pegasus/sites.v3/code.org/public/teacher-dashboard/review-hociyskvuwa/splat.haml

```diff
@@ -23,15 +23,24 @@ title: Review Hour of Code special events
     background: #f2dede;
   }
 
+-splat = request.splat_path_info[1..-1].split("/")
+-country = splat.first
+-state_code = splat.last unless country == splat.last
+
 %h1 Review Hour of Code special events
 
 -results = []
 -events = []
+
 -DB[:forms].where(kind:'HocSignup2015', review:nil).each do |i|
-  -results << JSON.parse(i[:data]).merge(secret: i[:secret])
+  -results << JSON.parse(i[:data]).merge(secret: i[:secret]).merge(JSON.parse(i[:processed_data]))
```

Comment:
@tanyaparker This is what gives the error you saw yesterday on your local, because your data hasn't been processed. I'll try adding a check for this